### DISCUSSION TITLE
What's the env var for a Github Token?

### DISCUSSION THREAD

**Discussion by kencieszykowski** [cooperspencer/gickup#184] [Q&A] [answered]
Hey all,

Stupid question here, but can't seem to find it in the documentation.  Long story short, I'm pushing this out to Kubernetes and would prefer to map the token as a k8s secret, instead of in a ConfigMap YAML.  

[Documentation Link ](https://cooperspencer.github.io/gickup-documentation/docs/configuration/source/github/)

```
token: your github token. You don't need one, if you backup only public repositories, although it is possible to hit the limit for the Github-API. It can also be an environment variable.
```

Downside is that I can't see any reference to what the expected environment variable should be.  Looking through code, my best guess is `tokenString`; but that doesn't seem to follow standard env var syntax (also the only other Env var I could see is `GICKUP_TIME_FORMAT`, so I'd expect something similar).

Thanks!

**Comment by cooperspencer** [comment, marked as answer]
Hi,
For the tokens you can use whatever name you want. For example, you can export your token as TOKEN_XYZ. If you reference the token in the config to TOKEN_XYZ, gickup will look up if it finds an environment variable with said name.

**Reply by kencieszykowski** [reply]
Yikes; I'm a dummy.  Thanks for the clarification here and ESPECIALLY thanks for the _incredibly_ quick reply!

**Reply by cooperspencer** [reply]
you are welcome.
